ignite/engine/events.py

```diff
@@ -4,50 +4,68 @@
 from enum import Enum
 import numbers
 import weakref
+from copy import deepcopy
```

Comment:
This is not used ?

Reply:
Correct. Forgot to remove this from a previous version